app/src/androidTest/java/com/willowtree/vocable/room/MigrationTest.kt

```diff
@@ -235,6 +237,7 @@ class MigrationTest {
                     phraseCursor.close()
 
                 Assert.assertTrue(phrasesAdded.contains(testPhraseV4))
+                Assert.assertEquals(0L, timestamp)
```

Comment:
v cool that the timestamp is initialized at 0L instead of null

Reply:
Yeah it's nullable for Kotlin reasons, because the db can return null, even if the field is non-nullable.  So I always make them nullable when I can so Kotlin forces you to handle it.